File: Logica_de_Programacao_Python/Desafio_12/desafio_12/Desafio_12.py
vetor = []

for i in range(10):
    valor = int(input(f"Digite o {i + 1}º valor: "))
    vetor.append(valor)

contador_impares = 0
for valor in vetor:
    if valor % 2 != 0:
        contador_impares += 1

print(f"\nQuantidade de valores ímpares no vetor: {contador_impares}")

File: Logica_De_Programacao_Java/Desafio_12/src/Desafio_12.java
import java.util.Scanner;

public class Desafio_12 {
    public static void main(String[] args) {

        Scanner entrada = new Scanner(System.in);
        int[] vetor = new int[10];
        int quantidadeImpares = 0;

        System.out.println("Digite 10 valores inteiros:");
        for (int i = 0; i < 10; i++) {
            System.out.print("Valor " + (i + 1) + ": ");
            vetor[i] = entrada.nextInt();
        }

        for (int i = 0; i < 10; i++) {
            if (vetor[i] % 2 != 0) {
                quantidadeImpares++;
            }
        }

        System.out.println("Quantidade de valores ímpares no vetor: " + quantidadeImpares);
        entrada.close();
    }
}